react/src/components/masthead/components/SprkMastheadSelector/SprkMastheadSelector.js

```diff
@@ -131,20 +136,32 @@ class SprkMastheadSelector extends Component {
         {isOpen && (
           <div
             className={classNames(
-              'sprk-c-Masthead__selector-dropdown sprk-c-Dropdown',
+              'sprk-c-Masthead__selector-dropdown',
               additionalClasses,
             )}
             data-sprk-dropdown="dropdown-selector"
           >
-            <div className="sprk-c-Dropdown__header">
+            <div className="sprk-c-Masthead__selector-dropdown-header">
               <SprkLink
                 variant="plain"
-                additionalClasses="sprk-o-Stack sprk-o-Stack--split@xxs sprk-o-Stack--center-column sprk-u-Width-100"
+                additionalClasses="
+                  sprk-o-Stack
+                  sprk-o-Stack--split@xxs
+                  sprk-o-Stack--center-column
+                  sprk-u-Width-100
```

Comment:
Stack has width: 100%; set on it, why is this !important width 100% util here?

Reply:
I checked, and it wasn't doing anything -- I'll take this out!

Reply:
Nice, glad you are gonna take it out! I still see it here in the code, did you forget to remove it?